src/renderer/assets/styles/global.scss

```diff
@@ -41,6 +41,23 @@ body {
 
 input {
   font-family: Nunito, sans-serif;
+
+  &[type="search"]::-webkit-search-cancel-button{
+    appearance: none;
```

Comment:
Isn't it just possible to style the system-level cross instead of eliminating it? Aren't we over-engineering the solution to this problem? This would probably solve the out of sync state problem with controlled components too (because the system-level cross is equivalent to the user setting the text to empty which is captured by the input's change event, as opposed to manipulating the React state which can cause inconsistencies)

Reply:
ah, great :)

https://github.com/edrlab/thorium-reader/commit/62c568d679be3f401de99c5a4281b597e4470fb4